src/facets/DiamondManagerFacet.sol

```diff
@@ -139,6 +140,20 @@ contract DiamondManagerFacet {
         emit SeasonStarted(s.currentSeasonId, _rewardTokenToDistribute);
     }
 
+    function changeBoostPoints(address[] memory addresses, uint256[] memory newBoostPoints) external onlyOwner {
+        if (addresses.length != newBoostPoints.length) revert DiamondManagerFacet_InvalidAddressReductionCombination();
+        uint256 currentSeasonId = s.currentSeasonId;
+
+        for (uint256 i; i < addresses.length; i++) {
+            UserData storage _userData = s.usersData[currentSeasonId][addresses[i]];
+            uint256 newBoostPoint = newBoostPoints[i];
+            if (_userData.depositAmount == 0) {
```

Comment:
Why will there be users with `0` deposit points in this?

Reply:
There'll be none. Better to have check I guess.

Reply:
In that case, it will be better to `revert` instead of `continue` because we would be doing something wrong. What do you think?

Reply:
sounds good